apps/emqx/include/logger.hrl

```diff
@@ -40,6 +40,21 @@
     end
 ).
 
+%% NOTE: do not forget to add every used event id to the default value of
+%% `log.thorttling.events_ids` list.
+-define(SLOG_THROTTLE(EventId, Level, Data),
+    ?SLOG_THROTTLE(EventId, Level, Data, #{})
+).
+
+-define(SLOG_THROTTLE(EventId, Level, Data, Meta),
```

Comment:
Maybe just use the one-word `msg` as `EventId` ?
Otherwise it's duplicated as `msg` and as `throttle_event_id`